draft/spec/index.md

```diff
@@ -1353,8 +1359,8 @@ A. Wright; H Andrews.20 September 2018. URL: <https://json-schema.org/latest/jso
 <span id="ref-oais"/>**\[OAIS]** Reference Model for an Open Archival Information System (OAIS), Issue 2. June 2012.
 URL: <https://public.ccsds.org/pubs/650x0m2.pdf>
 
-<span id="ref-ocfl-implementation-notes"/>**\[OCFL-Implementation-Notes]** OCFL Implementation Notes. URL:
-<https://ocfl.io/draft/implementation-notes>
+<span id="ref-ocfl-implementation-notes"/>**\[OCFL-Implementation-Notes]** OCFL Implementation Notes v1.1. URL:
+<https://ocfl.io/1.1/implementation-notes>
```

Comment:
Should we instead leave this as `draft`?

Reply:
Yes, fixed in [360e855](https://github.com/OCFL/spec/pull/615/commits/360e8556a4e75a0eaa7efb62641a159563c95318)